src/utils/webrtc/MediaDevicesManager.js

```diff
@@ -250,6 +253,23 @@ MediaDevicesManager.prototype = {
 			this._preferenceVideoInputList = newVideoInputList
 			BrowserStorage.setItem('videoInputPreferences', JSON.stringify(this._preferenceVideoInputList))
 		}
+
+		// If the preference list is the same as devices, it means user has not set any preferences yet
+		const audioPreferencesMatch = this.isPreferredDevice(devices, this._preferenceAudioInputList, 'audioinput')
+		if (audioPreferencesMatch) {
+			this.audioInputPreferred = false
+		}
+
+		const videoPreferencesMatch = this.isPreferredDevice(devices, this._preferenceVideoInputList, 'videoinput')
+		if (videoPreferencesMatch) {
+			this.videoInputPreferred = false
+		}
+	},
+
+	isPreferredDevice(devices, preferenceList, kind) {
+		const preferenceListIds = preferenceList.map(device => device.deviceId)
+		const deviceIds = devices.filter(device => device.kind === kind).map(device => device.deviceId)
+		return preferenceListIds.every((deviceId, index) => deviceId === deviceIds[index])
```

Comment:
Note: Using this method, it means that Devices section will be always shown first in two cases:
* One device exists for one of the streams
* User coincidentally chooses the first device as in `devices` order 

To solve that, we can track if `updatePreferences` is called ( e.g: in browserStorage` 'update-preferences' = true`) so we know that the device was once approved to be used by user. wdyt?